src/components/formio/columns/columns.component.tsx

```diff
@@ -1,11 +1,13 @@
-import {Component} from '@components';
 import {IComponentProps} from '@types';
 import clsx from 'clsx';
 import {ComponentSchema} from 'formiojs';
 import React from 'react';
 
+import {Component} from '../../utils';
```

Comment:
```suggestion
import {Component} from '@components/utils';
```
I find the relative paths so incredibly hard to reason about where exactly the thing is being imported from :grimacing:

Reply:
Not opposed to introducing a separate namespace for this but lets address this elsewhere.